### ISSUE TITLE
Map Style="Public Transport"

### ISSUE THREAD

**Issue by bjohas** [osmandapp/Osmand#3168]
I tried using "Configure map > Transport > tram/train/underground routes", and it's quite difficult in built-up areas (in large cities) as the routes are overshadowed by roads. This suggestion builds on https://github.com/osmandapp/Osmand/issues/3167 (fixing layers).

The tram/train/underground routes are rendered in different colours (at least in the cases I looked at, these correspond with the colours used on the official maps, which is helpful). However, because different types of roads also render in different colours, it's difficult to actually follow routes.

My suggestion is to add a "Map Style" ("Configure map > Map Style") in which all non-public-transport features are rendered in grey, leaving public transport routes standing out (as long as these are rendered ontop of non-public-transport features, as in #3167).

**Comment by bjohas** [comment]
Rather than "rendered in grey" I meant a simplified rendering, which uses a reduced set of colours (green for forests, blue for water, etc). See e.g. some of the metro/subway apps on Google play.

**Comment by sonora** [comment]
I guess the new "City map" style can fix this?

**Comment by xmd5a2** [comment]
Partially. It was not a primary goal.

**Comment by bjohas** [comment]
Yeah partially. The "city map" style is nice, but major roads are still coloured in darker/lighter brown, so the public transport routes does really stand out (as they do in the above example).

**Comment by bjohas** [comment]
Hello - just wondering whether you've had any more thoughts on this! Many thanks!

**Comment by bjohas** [comment]
A first step might be to have a map style=none? (Assuming that that would still show transport items, POIs etc). Would that be simple to do?

**Comment by bjohas** [comment]
Hello, I'm still very interested in this! Thanks!

**Comment by erik55** [comment]
Maybe this is a duplicate of #5230 ?

**Comment by prototype99** [comment]
this one is older so #5230 would have to be the duplicate. burying it in the different map styles may not be the best decision if there's no way to add a quick action to switch the map. if the transport is hourly and you wanted to quickly check it's the right one you wouldn't want to miss it because switching the map took too long.

**Comment by njam** [comment]
Maybe a simple _Road Style_ "basic" would do the trick, which would render all streets in white?

Compare OsmAnd:
![20191211_161304](https://user-images.githubusercontent.com/360800/70622539-0d858800-1c14-11ea-8ed9-5839be3df856.jpg)


To Google Maps:
![20191211_161244](https://user-images.githubusercontent.com/360800/70622501-f8105e00-1c13-11ea-83cb-08b9dcf94507.jpg)

**Comment by Friptick** [comment]
Agreed. Not just Google but now Organic Maps (based on OSM) is better than OsmAnd for public transport. It is especially true for metro and other rail. See comparisons.

Here is OsmAnd in Touring map style, supposedly the best for public transport, and everything else possible turned off:   

![par1](https://github.com/osmandapp/OsmAnd/assets/33609255/83ca4ec5-fe73-483c-a83c-72aed8e4e068)

And here is Organic Maps in default view:

![par2](https://github.com/osmandapp/OsmAnd/assets/33609255/0788d878-6314-4857-840a-749072ab7eec)

This is a problem for the OsmAnd project. Related: #12393 #10095 #17373 #18807

**Comment by pebogufi** [comment]
What about using  public transport using as an overlay ?
Then i could use the slider to change the visiblity of both
- the base map and
- public transport map
Everybody could choose how much he wants to see of each...
